RecoTracker/MkFitCore/interface/HitStructures.h

```diff
@@ -0,0 +1,874 @@
+#ifndef RecoTracker_MkFitCore_interface_HitStructures_h
+#define RecoTracker_MkFitCore_interface_HitStructures_h
+
+#include "RecoTracker/MkFitCore/interface/Config.h"
+#include "RecoTracker/MkFitCore/interface/Hit.h"
+#include "RecoTracker/MkFitCore/interface/Track.h"
+#include "RecoTracker/MkFitCore/interface/TrackerInfo.h"
+
+#include <algorithm>
+#include <array>
+#include "tbb/concurrent_vector.h"
+
+//#define DUMPHITWINDOW
+
+namespace mkfit {
+
+  class IterationParams;
+
+  typedef tbb::concurrent_vector<TripletIdx> TripletIdxConVec;
+
+  // for each layer
+  //   Config::nEtaBin vectors of hits, resized to large enough N
+  //   filled with corresponding hits
+  //   sorted in phi
+  //   and with a corresponding BinInfoSomething
+  //
+  // My gut feeling is that we could have them all in one place and just store
+  // indices into small enough eta-phi bins (oh, and eta is a horrible separation variable)
+  // 10*50000*64/1024/1024
+  //    30.51757812500000000000
+  // Or make them smaller ... if we could use short indices that might make a difference.
+  //
+  // Need to add eta/phi ... or something significantly better to the Hit struct
+  //
+  // At least ... fast eta/phi estimators.
+  //
+  // Oh, and we should use radix sort. Could one vectorize this?
+
+  // Need a good "array of pods" class with aligned alloc and automatic growth.
+  // For now just implement the no-resize / no-destroy basics in the BoH.
```

Comment:
please clarify what is still relevant here (in an issue perhaps), and remove the rest.

Reply:
Addressed in Round 2 follow-up PR trackreco/cmssw#76 commit https://github.com/trackreco/cmssw/pull/76/commits/d2c6a5ba4edbe357152710835beb41fb16faf51e